guides/RUNNING-BENCH-TPS.md

```diff
@@ -0,0 +1,142 @@
+# Running Solana Bench TPS on SVMKit Network
+
+This guide demonstrates how to install and run `solana-bench-tps` on a SVMKit network for benchmarking transaction processing capabilities. Benchmarking is essential for validators and cluster operators to evaluate their cluster's performance.
+
+## Overview
+
+This guide covers:
+1. Setting up the required environment
+2. Building the Solana benchmarking tool
+3. Running benchmark tests with various configurations
+4. Interpreting benchmark results
```

Comment:
Guide doesn't cover building solana benchmark anymore but does show how to fetch it from apt.